### DISCUSSION TITLE
Alternative DSL file on Structiruzr

### DISCUSSION THREAD

**Discussion by cirof-vr** [structurizr/lite#141] [Q&A]
Hi!

Is there a way to configure Structurizer to choose a different DSL file instead of "workspace.dsl" and "workspace.json"?

Tks in advance...

**Comment by simonbrowndotje** [comment]
Yes, see https://docs.structurizr.com/lite/usage#workspace-filename for details of how to do this.

**Comment by cirof-vr** [comment]
Good morning!

It worked.

Thanks a lot!


Att,

***@***.***<https://www.vr.com.br/>    ***@***.***
Ciro Rezende Formenton
Arquitetura de Soluções
<https://pt-br.facebook.com/VRBeneficios/>   <https://www.instagram.com/vrbeneficios/>    <https://twitter.com/vrbeneficios>    <https://www.youtube.com/user/vrprodutos>    <https://www.linkedin.com/company/vrbeneficios/>
(11) 99949-7445
***@***.******@***.***>



De: Simon Brown ***@***.***>
Enviada em: sábado, 4 de maio de 2024 06:19
Para: structurizr/lite ***@***.***>
Cc: Ciro Rezende Formenton ***@***.***>; Author ***@***.***>
Assunto: Re: [structurizr/lite] Alternative DSL file on Structiruzr (Discussion #141)

Você não costuma receber emails de ***@***.******@***.***>. Saiba por que isso é importante<https://aka.ms/LearnAboutSenderIdentification>

Yes, see https://docs.structurizr.com/lite/usage#workspace-filename for details of how to do this.
